application_form/tests/api/test_apartment_reservation_api.py

```diff
@@ -209,9 +212,39 @@ def test_root_apartment_reservation_detail_installment_candidates(
         "due_date": None,
     }
 
+    # Test two flexible payments, the first one being rounded down and the second one
+    # rounded up to the nearest cent.
+    installment_template_3.percentage_specifier = (
+        InstallmentPercentageSpecifier.SALES_PRICE_FLEXIBLE
+    )
+    installment_template_3.save()
+    response = sales_ui_salesperson_api_client.get(
+        reverse(
+            "application_form:sales-apartment-reservation-detail",
+            kwargs={"pk": reservation.id},
+        ),
+        format="json",
+    )
+    assert response.status_code == 200
+
+    installment_candidates = response.data["installment_candidates"]
+    assert len(installment_candidates) == 5
+    assert installment_candidates[2] == {
+        "type": installment_template_3.type.value,
+        "amount": 5555555,
+        "account_number": installment_template_3.account_number,
+        "due_date": None,
+    }
+    assert installment_candidates[3] == {
+        "type": installment_template_4.type.value,
+        "amount": 5555556,
+        "account_number": installment_template_4.account_number,
+        "due_date": None,
+    }
+
 
 @pytest.mark.django_db
-def test_contract_pdf_creation_unauthorized(user_api_client):
+def test_contract_pdf_creation_unauthorized(elasticsearch, user_api_client):
```

Comment:
All of these elasticsearch fixture additions seem to be completely unrelated to the flexible installment thing, so it would be nicer if these were in their own commit.

Reply:
Oh yeah, absolutely, I may have squashed this accidentally. I'll separate it. The main point here was that the `setup_elasticsearch` and `teardown_elasticsearch` are skipped without using any ES fixture. Just for the sake of consistency as  the tests will leave the test index dangling without it.